src/components/components/ebay-dialog-base/component.js

```diff
@@ -53,10 +61,18 @@ module.exports = {
         this.windowEl = this.getEl('window');
         this.closeEl = this.getEl('close');
         this.bodyEl = this.getEl('body');
-        this.transitionEls = [this.windowEl, this.rootEl];
+        if (this.input.transitionEl === 'root') {
+            this.transitionEls = [this.rootEl];
+        } else {
+            this.transitionEls = [this.windowEl, this.rootEl];
+        }
         // Add an event listener to the dialog to fix an issue with Safari not recognizing it as a touch target.
         this.subscribeTo(this.rootEl).on('click', () => {});
 
+        // Added to document in case the object is not a modal type
+        // This is so we can escape even if focus is not present on popup
+        document.addEventListener('keydown', this.handleKeydown.bind(this));
```

Comment:
Since we are finally using `marko@4` how do you feel about trying out the `<subscribe>` tag here? (https://github.com/marko-js/tags/tree/master/tags/subscribe).

```marko
<subscribe to=document on-keydown("handleKeydown")/>
```

I bring this up because what is currently in this PR can cause a bug since it is never cleaning up the event listener.